tools/olm-bundle/generate-olm-bundle.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+#
+# Generates OLM bundle using existing CRDs
+#
+#
+source ../multi-platform-support.sh
+
+PACKAGE_NAME=strimzi-cluster-operator
+VERSION=$1
+
+CHANNELS=latest
+PREVIOUS_BUNDLE_VERSION=$(curl -s https://raw.githubusercontent.com/operator-framework/community-operators/master/community-operators/strimzi-kafka-operator/strimzi-kafka-operator.package.yaml | yq e '.channels[0].currentCSV' -)
+
+CSV_TEMPLATE_DIR=./csv-template
```

Comment:
Actually, I think you need to use `$(dirname $(realpath $0))/` for all the paths.